<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>manager_window</class>
 <widget class="QMainWindow" name="manager_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1168</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1181</width>
      <height>371</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>管理图书</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>811</width>
        <height>311</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>135</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>19</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>19</number>
      </attribute>
      <column>
       <property name="text">
        <string>Book id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Book name</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>writer</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>press</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>storage</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>0</y>
        <width>120</width>
        <height>34</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Al Tarikh</family>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="text">
       <string>manage books</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>270</y>
        <width>331</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>281</width>
        <height>224</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <family>Al Tarikh</family>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="text">
              <string>book id</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <family>Al Tarikh</family>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="text">
              <string>book name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <family>Al Tarikh</family>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="text">
              <string>writer</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <family>Al Tarikh</family>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="text">
              <string>press</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="font">
              <font>
               <family>Al Tarikh</family>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="text">
              <string>price</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <family>Al Tarikh</family>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="text">
              <string>storage</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLineEdit" name="find_bookid"/>
           </item>
           <item>
            <widget class="QLineEdit" name="find_bookname"/>
           </item>
           <item>
            <widget class="QLineEdit" name="find_writer"/>
           </item>
           <item>
            <widget class="QLineEdit" name="find_press"/>
           </item>
           <item>
            <widget class="QLineEdit" name="find_price"/>
           </item>
           <item>
            <widget class="QLineEdit" name="find_storage"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="find">
           <property name="text">
            <string>find</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="add">
           <property name="text">
            <string>add</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>281</width>
        <height>34</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <family>Al Tarikh</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>book id:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="delete_book_id"/>
       </item>
       <item>
        <widget class="QPushButton" name="delete_book">
         <property name="text">
          <string>delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>用户管理</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>301</x>
        <y>10</y>
        <width>781</width>
        <height>301</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="font">
            <font>
             <family>Al Tarikh</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>students</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="fresh_borrow">
           <property name="text">
            <string>更新</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget_2">
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>197</number>
         </attribute>
         <column>
          <property name="text">
           <string>stu name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>stu id</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>stu grade</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>password</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>261</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="font">
          <font>
           <family>Al Tarikh</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>delete student</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>stu name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="return_bookid"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="ret">
           <property name="text">
            <string>delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>借书管理</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>1121</width>
        <height>321</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>140</number>
      </attribute>
      <column>
       <property name="text">
        <string>loan id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>book id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>book name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>storage</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>stu name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>borrow time</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>return time</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
